### PR TITLE
Var decl gaps

### PR DESCRIPTION
**Pull request by DomHenderson**
Handles variable declaration statements with gaps on the lhs

### CODE REVIEW

File: src/cairoWriter.ts [hunk 1]
```diff
@@ -206,10 +207,9 @@ class VariableDeclarationStatementWriter extends CairoASTNodeWriter {
 
     const documentation = getDocumentation(node.documentation, writer);
     const declarations = node.assignments.flatMap((id) => {
-      if (id === null)
-        throw new NotSupportedYetError(
-          `VariableDeclarationStatements with gaps not implemented yet`,
-        );
+      if (id === null) {
+        return [`__warp_gv${this.gapVarCounter++}`];
+      }
```
Comment (GuyMcComb) [inline src/cairoWriter.ts:212]: Out of interest, where would there be an empty variable declaration?

Comment (DomHenderson) [reply src/cairoWriter.ts:212]: Usually something like `(uint a,) = f()` where the person doesn't care about the second return from f

Comment (GuyMcComb) [reply src/cairoWriter.ts:212]: I thought these would have been handled by the tuple filler? or was that only for assignments and not vardecl statements?

Comment (DomHenderson) [reply src/cairoWriter.ts:212]: Yep, they aren't actually TupleExpressions, they're just VariableDeclarationStatements with more slots than children

Comment (GuyMcComb) [reply src/cairoWriter.ts:212]: HA ok cool.

**Review by GuyMcComb — APPROVED** [review]

**Pull request merged**